source/atlas-cli-local-cloud.txt

```diff
@@ -0,0 +1,90 @@
+.. _atlas-cli-local-cloud:
+
+=========================================================
+Manage Local and Cloud Deployments from the {+atlas-cli+}
+=========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the {+atlas-cli+} to work with |service|, including using |fts| 
+and {+avs+}, throughout the entire software development lifecycle, from 
+your local environment to the cloud.
+
+.. include:: /includes/fact-atlas-deployments-preview.rst
+
+With the ``atlas deployments`` commands, you can perform the following
+actions:
+
+- Create and manage local |service| deployments. Local |service| 
+  deployments reside on your computer and provide a non-production 
+  environment for development.
+- Create and manage |service| cloud deployments. Cloud |service| 
+  deployments reside in the cloud for non-production and production use.
+- Manage |fts| and {+avs+} indexes for your local and cloud |service| 
+  deployments.
+
+.. note::
+
+   For local |service| deployments, you can manage only deployments that
+   you created with the ``atlas deployments`` command. If you created a 
+   local|service| deployment without using the {+atlas-cli+}, you can't
+   manage that deployment with the {+atlas-cli+}.
+
+You can use only ``atlas deployments`` {+atlas-cli+} commands with 
+local |service| deployments. You can use all {+atlas-cli+} commands, 
+including ``atlasdeployments`` commands,  with cloud |service| 
+deployments.
+
+Supported Actions
+-----------------
+
+You can perform the following actions with the ``atlas deployments`` 
+command  for local and cloud |service| deployments including, but not limited to:
+
+- **Set up** a new |service| deployment.
+   
+- **List** |service| deployments.
+  
+- **Connect** with {+mongosh+} or :compass:`Compass </>`, or return the 
+  connection string for an |service| deployment.
+
+- **Pause or start** an |service| deployment.
+  
+- **Delete** an |service| deployment.
+  
+- **Access logs** for an |service| deployment.
+  
+- **Create or manage |fts| and {+avs+} indexes** to run a full-text 
```

Comment:
doesn't look like this is rendering properly
<img width="795" alt="image" src="https://github.com/mongodb/docs-atlas-cli/assets/34902148/d4a26644-706c-45f6-b07c-89a2ad5ed7e8">

Reply:
fixed